Use parameters/props in cypher create/set queries

prasanthu opened the pull request.

prasanthu committed:
Use parameters/props in cypher create/set queries

prasanthu committed:
Merge branch 'develop' into feature/neo4j-cypher-errors

prasanthu committed:
Remove unused line

prasanthu committed:
Remove unused line

prasanthu committed:
Merge branch 'develop' into feature/neo4j-cypher-errors

prasanthu committed:
Update Schema.kt

Remove unused logger

prasanthu committed:
Merge branch 'develop' into feature/neo4j-cypher-errors

vihangpatil commented:
@prasanthu 
The original code sets every property value in string.  Does the new logic change that?  Will it break for other classes which have non-string fields.

kmmatwork reviewed (approved):
OK from me!

prasanthu commented:
> @prasanthu
> The original code sets every property value in string. Does the new logic change that? Will it break for other classes which have non-string fields.

Looking into this.

prasanthu committed:
Change the property value type to String

Our previous implementaion of create and set methods were treating
all values as string. This patch will retain that behaviour while
using the parameter style in cypher queries

prasanthu committed:
Merge branch 'develop' into feature/neo4j-cypher-errors

prasanthu committed:
Merge branch 'develop' into feature/neo4j-cypher-errors

prasanthu committed:
Simplify conversion
